astropy/wcs/wcs.py

```diff
@@ -1106,12 +1106,12 @@ def _read_sip_kw(self, header, wcskey=""):
         if a is None and b is None and ap is None and bp is None:
             return None
 
-        if str("CRPIX1") not in header or str("CRPIX2") not in header:
+        if str("CRPIX1"+wcskey) not in header or str("CRPIX2"+wcskey) not in header:
```

Comment:
For consistency with the coding notation used in the rest of this module, it might be better to do ``"CRPIX1{0}".format(wcskey)``